docs/features/kubernetes/authentication.md

```diff
@@ -0,0 +1,35 @@
+---
+id: kubernetes-auth
```

Comment:
Since your link in the configuration page has a path like `/docs/features/kubernetes/authentication`, I think this needs to be
```suggestion
id: authentication
```

Reply:
I was not sure whether I could, didn't want the id to collide with any other in the `Auth and identity` section, seems `authentication` id is not in use so I'll apply this.